My_project/Task_3.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>task_3</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,700&family=Roboto:wght@300&display=swap" rel="stylesheet">
+   <style> td{
+        padding: 7px;
+   }
+   .Link_1{
```

Comment:
Your class names are not informative. 
Will be better for example: .link_button, .button, .primary_button, .secondary_button and etc.

Reply:
done